socketshark/session.py

```diff
@@ -86,6 +86,18 @@ def __init__(self, shark, client, info={}):
         msg.update(subscription.extra_data)
         await self.send(msg)
 
+    async def send_unsubscribe(self, subscription, data=None, error=None):
+        msg = {
+            'event': 'unsubscribe',
```

Comment:
Should this be `unsubscribed` since we are notifying the client of something the server did versus sending a request to unsubscribe.

Reply:
I was thinking it should be the same name as for a user-triggered unsubscribe. Basically the client code can assume that if an `unsubscribe` event happens the subscription was unsubscribed. Having two different events is a bit confusing and requires clients to implement both. Do you see any issues with clients not being able to uniquely distinguish unsubscribe types?